lib/matplotlib/layout_engine.py

```diff
@@ -242,6 +249,9 @@ def set(self, *, h_pad=None, w_pad=None,
             If h/wspace < h/w_pad, then the pads are used instead.
             Default to :rc:`figure.constrained_layout.hspace` and
             :rc:`figure.constrained_layout.wspace`.
+        rect : [l, b, w, h]
+            Rectangle in figure coordinates to perform constrained layout in
+            [left, bottom, width, height], each from 0-1.
```

Comment:
```suggestion
        rect : tuple of 4 floats
            Rectangle in figure coordinates to perform constrained layout in
            (left, bottom, width, height), each from 0-1.
```

Reply:
Fixed all these, also in `TightLayoutEngine`..